kore/src/Kore/Repl.hs

```diff
@@ -417,12 +439,35 @@ replInterpreter =
     unparseStrategy :: CommonStrategyPattern level -> String
     unparseStrategy =
         \case
-            Bottom -> "Reached goal!"
+            Bottom -> "Reached bottom"
             Stuck pat -> "Stuck: \n" <> unparseToString pat
             RewritePattern pat -> unparseToString pat
 
-    indexNotFound :: String
-    indexNotFound = "Variable or index not found"
+    formatSourceAndLocation :: (Source, Location) -> Maybe String
```

Comment:
Maybe we could give instances of `Pretty` for `Source` and `Location` instead?

Reply:
The problem with that is they are not really useful on their own, especially `Location`. Ideally, I would change this attribute in the frontend to be a single attribute, although I'm not really sure it's worth the effort since it's really not that important.

Reply:
We could make it a single attribute in the backend.

Reply:
I thought about that, but don't we parse attributes sequentially? So when trying to merge two attributes, you'd have to "guess" the order? Or maybe I am missing something, because the naive combined parser fails:

```
data SourceLocation = SourceLocation
    { location :: !Location
    , source   :: !Source
    } deriving (Eq, Ord, Show, Generic)

instance ParseAttributes SourceLocation where
    parseAttribute patt attrs =
        SourceLocation
            <$> parseAttribute patt def
            <*> parseAttribute patt def
```

Reply:
Nevermind, that was obviously the wrong approach. I figured out how to do it and updated the PR (see latest commit).